### PR TITLE
arDominionB5Plugin: update advanced search forms

### PR DESCRIPTION
**Pull request by sevein**
New responsive layout based on Bootstrap 5.

The help message is implemented as a [tooltip](https://getbootstrap.com/docs/4.0/components/tooltips/).

## informationobject/browse

image

## actor/browse

image

### CODE REVIEW

File: plugins/arDominionB5Plugin/js/advancedSearch.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+  "use strict";
+
+  $(() => $(".adv-search").each((_, el) => new AdvancedSearch(el)));
+
+  class AdvancedSearch {
+    constructor(element) {
+      this.$element = $(element);
+      this.$form = this.$element.find('form[name="advanced-search-form"]');
+      this.$reposFacet = this.$element
+        .find("#\\#facet-repository")
```
Comment (jraddaoui) [inline plugins/arDominionB5Plugin/js/advancedSearch.js]: Please, check this is working as noted in sync time.

File: plugins/arDominionB5Plugin/modules/search/templates/_advancedSearch.php [hunk 2]
```diff
@@ -150,116 +167,99 @@
...
-                <?php if ($showMaterial) { ?>
-                  <div class="filter<?php echo $showCopyright ? '-right' : ''; ?>">
-                    <?php echo $form->materialType
-                        ->label(__('General material designation'))
-                        ->renderRow(); ?>
+              <div class="col-12">
+                <div class="d-grid d-sm-block mt-2">
+                  <div class="form-check d-inline-block me-2">
+                    <input class="form-check-input" type="radio" name="topLod" id="adv-search-top-lod-1" value="1" <?php echo $topLod ? 'checked' : ''; ?>>
+                    <label class="form-check-label" for="adv-search-top-lod-1"><?php echo __('Top-level descriptions'); ?></label>
```
Comment (jraddaoui) [inline plugins/arDominionB5Plugin/modules/search/templates/_advancedSearch.php:209]: Use fieldset and legend ...

https://www.w3.org/WAI/tutorials/forms/grouping/

Or radiogroup role ...

https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/radio/radio-1/radio-1.html

**Review by jraddaoui — APPROVED** [review]
Awesome!!

File: plugins/arDominionB5Plugin/js/advancedSearch.js (hunk 1 above)
Comment (sevein) [reply plugins/arDominionB5Plugin/js/advancedSearch.js]: Fixed!

File: plugins/arDominionB5Plugin/modules/search/templates/_advancedSearch.php (hunk 2 above)
Comment (sevein) [reply plugins/arDominionB5Plugin/modules/search/templates/_advancedSearch.php:209]: Thanks!

**Pull request merged**